tests/unit/test_file_operations.py

```diff
@@ -88,6 +98,30 @@ def test_append_to_file(self):
 
         self.assertEqual(content_after, append_text + append_text)
 
+    def test_update_file(self):
+        with open(self.test_nested_file, "r") as f:
+            old_content = f.read()
+
+        update_file(self.test_nested_file, "test", "create")
```

Comment:
This interferes with the other tests, please isolate to prevent side-effects

Reply:
Will rewrite to isolate. how do i create an extra file for testing purposes? not that known with pytest and such yet

Reply:
I don't believe you need to create a new file - you just need to put it into a new function in the same file. Leverage the function pattern in the other functions so pytest knows the scope to run at.